We distribute the generated files in localedef package. Thus we need
to add the copyright header to the files

--- a/tools/java/org/unicode/cldr/posix/GenerateCharmap.java
+++ b/tools/java/org/unicode/cldr/posix/GenerateCharmap.java
@@ -102,6 +102,8 @@
         out.println("# charset:\t" + codeset);
         out.println("######################");
         out.println("#################################################################################################");
+        out.println("# Copyright (c) 2010, 2018, Oracle and/or its affiliates. All rights reserved.                  #");
+        out.println("#################################################################################################");
         out.println("# Copyright 1991-2011 Unicode, Inc. All rights reserved. Distributed under the Terms of Use in  #");
         out.println("# http://www.unicode.org/copyright.html.                                                        #");
         out.println("#                                                                                               #");
--- a/tools/java/org/unicode/cldr/posix/POSIXLocale.java
+++ b/tools/java/org/unicode/cldr/posix/POSIXLocale.java
@@ -174,6 +174,11 @@
         out.println("* see http://www.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap07.html                    *");
         out.println("* Locale Name : " + locale_name + "   Codeset : " + codeset);
         out.println("*************************************************************************************************");
+        out.println("* Copyright (c) 2010, 2018, Oracle and/or its affiliates. All rights reserved.                  *");
+        out.println("*                                                                                               *");
+        out.println("* The Unicode Data Files have been modified by Oracle.                                          *");
+        out.println("*                                                                                               *");
+        out.println("*************************************************************************************************");
         out.println("* Copyright 1991-2013 Unicode, Inc. All rights reserved. Distributed under the Terms of Use in  *");
         out.println("* http://www.unicode.org/copyright.html.                                                        *");
         out.println("*                                                                                               *");
